.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "public/lib/jsroot"]
+	path = public/lib/jsroot
+	url = https://github.com/root-project/jsroot
```

Comment:
we should have some good versioning, better to use specific branches, see https://git-scm.com/book/en/v2/Git-Tools-Submodules

check if and option could be added for shallow clone

Reply:
```shallow = true```